### ISSUE TITLE
problem when aligning photos to create 3d model of teeth

### ISSUE THREAD

**Issue by PandaBo25** [alicevision/Meshroom#2596]
Hi all, I am trying to create a 3D model of teeth from a video/images taken intra-orally. I am experiencing some problems with aligning photos, I think it is because the camera moves around the mouth and also the teeth are not fixed in position. I have tried several software and also building my own code in python, but the results were in all cases not satisfactory. If you have any tips and suggestions, it is highly appreciated! Thank you :)

I am using Mac.

**Comment by fabiencastan** [comment]
Yes, it's not possible to use standard photogrammetry, if the mouth is not maintained fixed.
Difficult to provide tips without seeing the images.
You can contact me privately if you cannot share the images publicly.

**Comment by PandaBo25** [comment]
[upper jaw images  Kopie.zip](https://github.com/user-attachments/files/17951013/upper.jaw.images.Kopie.zip)
Hi, thanks for your reply! I have attached the fotos. The teeth are kind of fix because of the box that was used for taking the images.
